docs/dev-manuals/architecture/user-journeys.md

```diff
@@ -0,0 +1,242 @@
+        Receive refund : 4
+```
+
+## Writing a review
+
+```mermaid
+journey
+    title Writing a review
+    section Enter online shop
+        Visit website : 4
+        Log in : 3
+    section Go to ordered item
+        Go to my orders : 3
+        Go to order in question : 3
+        Select item to review : 3
+    section Write review
+        Choose title : 4
+        Write text : 4
+        Submit review : 5
+```
+
+## Writing a review (Alternativ 1)
+
+```mermaid
+journey
+    title Writing a review (Alternativ 1)
+    section Enter online shop
+        Visit website : 4
+        Log in : 3
+    section Go to product
+        Navigate to product : 4
+        Go to Reviews section : 4
+        Click on "Review" : 4
+    section Write review
+        Choose title : 4
+        Write text : 4
+        Submit review : 5
+```
+
+## Looking for a specific product
+
+```mermaid
+journey
+    title Looking for a specific product
+    section Enter online shop
+        Visit website : 4
+        Log in (optional) : 3
+    section Look for specific product
+        Check out categories : 4
+        Go to promising category : 4
+        Apply filter to category : 4
+        Scroll through products : 4
+        Go to promising product : 5
+        Read product description : 4
+        Check out product variants : 4
+        Read reviews : 4
+        Decide if product found : 4
+```
+
+## Looking for a specific product (Alternative 1)
+
+```mermaid
+journey
+    title Looking for a specific product (Alternative 1)
+    section Enter online shop
+        Visit website : 4
+        Log in (optional) : 3
+    section Look for specific product
+        Go to search : 4
+        Search with text input : 4
+        Apply filter to search results : 4
+        Scroll through (filtered) search results : 4
+        Go to promising product : 5
+        Read product description : 4
+        Check out product variants : 4
+        Read reviews : 4
+        Decide if product found : 4
+```
+
+## Adding a new wishlist
+
+```mermaid
+journey
+    title Adding a new wishlist
+    section Enter online shop
+        Visit website : 4
+        Log in : 3
+    section Add wishlist
+        Go to my wishlists : 4
+        Click on "Add wishlist" : 4
+        Choose name : 4
+        Confirm creation : 5
+```
+
+## Adding an item to the wishlist
+
+```mermaid
+journey
+    title Adding an item to the wishlist
+    section Enter online shop
+        Visit website : 5
+        Log in (optional) : 3
+    section Browse online shop
+        Scroll through start page : 5
+        Check out categories : 3
+        Go to category : 3
+        Apply filter to category : 3
+        Scroll through products : 3
+        Go to product : 5
+        Read review : 1
+        Read another review : 5
+        Check out product variants : 3
+        Click on "Add to wishlist" : 5
+        Choose wishlist : 5
+```
+
+## Adding a new product that only has one variant
```

Comment:
One emoticon is placed above the diagram. The legend shows "4" and "Employee" as the labels, is there a way to make it clearer?

Reply:
This means that the user in this case is not the _standard_ user meaning a person shopping online in our store but the user in this case is an employee of the online shop. For each journey step of a user's journey, one can set a score for how happy the user is during that step. In many cases this is absolutely not clear and I tried my best to somehow make the changes of emotions (scores) during a journey as reasonable as possible. For instance, login always causes the score to go down a point because the user has a goal (e.g., add a new product) and is excited (positively which leads to a high score like 4 or 5) and they visit the website and then "Oh, I have to login, sh*t".